app/javascript/packs/OAS3Autocomplete.js

```diff
@@ -20,22 +20,36 @@ const addAutocompleteToParam = (param: Param, accountData: AccountData): Param =
   const xDataKey = param[X_DATA_ATTRIBUTE]
   const autocompleteData = accountData[xDataKey]
   const paramHasAutocompleteData = autocompleteData && autocompleteData.length > 0 &&
-   autocompleteData.every(param => param.name !== '')
+    autocompleteData.every(param => param.name !== '')
 
   return paramHasAutocompleteData
     ? {
       ...param,
       examples: autocompleteData.reduce((examples, item) => (
-        [...examples, {summary: item.name, value: item.value}]
-      ), [{summary: X_DATA_PARAMS_DESCRIPTIONS[xDataKey], value: '-'}])
+        [...examples, { summary: item.name, value: item.value }]
+      ), [{ summary: X_DATA_PARAMS_DESCRIPTIONS[xDataKey], value: '-' }])
     }
     : param
 }
 
-const getPathParameters = (path) => {
-  const hasParamsInGet = path.get && path.get.parameters
-  return hasParamsInGet ? path.get.parameters : path.parameters
-}
+const getPathParameters = (path) => (
+  Object.keys(path).reduce((params, item) => {
+    params[item] = path[item].parameters
+    return params
+  }, {})
+)
+
+const injectParametersToPath = (path, pathParameters, accountData) => (
+  Object.keys(pathParameters).reduce((parameters, item) => {
+    parameters[item] = {
+      ...path[item],
+      parameters: pathParameters[item]
+        ? pathParameters[item].map(param => X_DATA_ATTRIBUTE in param ? addAutocompleteToParam(param, accountData) : param)
+        : []
+    }
+    return parameters
+  }, {})
+)
```

Comment:
```suggestion
const injectParametersToPath = (path, pathParameters, accountData) => (
  Object.keys(pathParameters).reduce((parameters, item) => (
     { ...parameters, [item]: path[item], 
        parameters: pathParameters[item]
          ? pathParameters[item].map(param => X_DATA_ATTRIBUTE in param ? addAutocompleteToParam(param, accountData) : param)
        : []
    }, {})
)
```

You could extract also to a separate fn the pathParameters mod